ethcore/builtin/src/lib.rs

```diff
@@ -219,6 +226,10 @@ impl ModexpPricer {
 			x => (x * x) / 16 + 480 * x - 199_680,
 		}
 	}
+
+	fn mult_complexity_new(x: u64) -> u64 {
+		((x / 64) + if x % 64 == 0 { 0 } else { 1 }) ^ 2
```

Comment:
Replace `64` with `8` to represent limbs in bytes rather than bits per most recent EIP update